Source/UnitTests/Common/BitFieldTest.cpp

```diff
@@ -128,20 +128,20 @@ TEST(BitField, Assignment)
     object.full_u64 = val;
     EXPECT_EQ(val, object.full_u64);
 
-    object.full_s64 = (s64)val;
```

Comment:
Most of these are not "functional style" casts. I guess it would just be called a c-style cast.
The commit message isn't very accurate.